miner/worker.go

```diff
@@ -1068,35 +1068,39 @@ func (w *worker) commitWork(interrupt *atomic.Int32, timestamp int64) {
 	if err != nil {
 		return
 	}
-	// Fill pending transactions from the txpool into the block.
```

Comment:
Oh yeah, didn't think about that

Reply:
Hmm from what I can tell the pending transaction subscription goes via the txpool not the miner

Reply:
Interesting point about pending logs. I think it's fair to remove them since they will be able to get this on demand via multicall.

Reply:
They can fetch list of pending transactions and pass them to multicall. Tho yes there will be no filtering on the server-side. But to your point, yes they will not know which txes are more likely to be mined. Because we simply return a flat list of transactions as opposed to a weighted list. Perhaps that's something that can be changed?